ssn/chapters/OMS.html

```diff
@@ -1029,7 +1039,8 @@ <h4>Property Alignments - OMS focus</h4>
           ,<br> <a href="#SOSArelatedObservation">sosa:relatedObservation</a>
           ,<br> <a href="#SOSAusedForExecution">sosa:usedForExecution</a>
           ,<br> <a href="#SOSAwasObservedBy">sosa:wasObservedBy</a>
```

Comment:
Similar changes are needed on the lines above.
```suggestion
          <a href="#SOSArelatedObservation">sosa:relatedObservation</a>,
          <a href="#SOSAusedForExecution">sosa:usedForExecution</a>,
          <a href="#SOSAwasObservedBy">sosa:wasObservedBy</a>,
```